packages/lib/server-only/document/update-title.ts

```diff
@@ -0,0 +1,20 @@
+'use server';
+
+import { prisma } from '@documenso/prisma';
+
+export type UpdateTitleOptions = {
+  userId?: number;
```

Comment:
This is a mandatory option and should be reflected in the prisma query as well 😄